README.md

```diff
@@ -27,6 +28,7 @@ Want to test out new features and get fixes before everyone else? Install the
 beta channel to get access to early builds of Desktop:
 
  - [macOS](https://central.github.com/deployments/desktop/desktop/latest/darwin?env=beta)
+ - [macOS (M1 Chip)](https://central.github.com/deployments/desktop/desktop/latest/darwin-arm64?env=beta)
```

Comment:
```suggestion
 - [macOS (Apple Silicon)](https://central.github.com/deployments/desktop/desktop/latest/darwin-arm64?env=beta)
```

I tend to think this is the "best way" to highlight these things, I think a future Mac could very well be shipping with an M2 or M1X (or so my sources say)

Reply:
Good point! I chose `M1` because that's how [Apple showcases it](https://www.apple.com/mac/m1/) and I don't see any references to `Apple Silicon`, but I agree it's probably more accurate and future-proof 😄 Thanks!!